internal/worker/httpserverargs/manifold_test.go

```diff
@@ -50,12 +47,8 @@ func TestManifoldSuite(t *testing.T) {
 }
 
 func (s *ManifoldSuite) SetUpTest(c *tc.C) {
-	s.StateSuite.SetUpTest(c)
-
 	s.clock = testclock.NewClock(time.Time{})
-	s.stateTracker = stubStateTracker{
-		pool: s.StatePool,
-	}
+	s.stateTracker = stubStateTracker{}
```

Comment:
todo: Remove the stateTracker, don't just stub it out.

same in internal/worker/httpserver

Reply:
Will be done in another ticket.